funnel/translations/hi_IN/LC_MESSAGES/messages.po

```diff
@@ -2303,194 +2307,194 @@ msgstr "नाम:</strong> %(name)s"
 msgid "Username:</strong> %(username)s"
 msgstr "यूजरनेम:</strong> %(username)s"
```

Comment:
Either opening `<strong>` tag is missing, or the closing tag is incorrectly included.

Reply:
This problem remains.